### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by raandree** [aldrichtr/infraspective#100] [General]
Thanks for continuing the work from the [Operation-Validation-Framework](https://github.com/PowerShell/Operation-Validation-Framework) project. I tried to get started and prepare a demo but found it difficult.

Are you going to release [infraspective](https://github.com/aldrichtr/infraspective) on the PowerShell Gallery so that only calling `Install-Module` is required to download all the required bits?

**Comment by aldrichtr** [comment]
Thank you for your interest in infraspective!  Yes, I am planning on releasing it as a module to the gallery.  I had to focus on another assignment over the past few weeks, but I plan to get back to this project this week.

**Reply by raandree** [reply]
It would be nice if the documentation included a "Getting Started" section. Getting started with the functionality was not easy for me.

**Reply by aldrichtr** [reply]
That is a good point.  I will definitely add that.

**Comment by aldrichtr** [comment]
@raandree 
I added a `Getting Started` section to the readme, in pr #102   

I am working on additional documentation such as the syntax for the audit files, configuration and output settings, as well as refreshing the project's roadmap and issues.  Please let me know if the instructions are helpful...
